src/site/_includes/macros/shows-card.njk

```diff
@@ -7,34 +7,19 @@
 {% set description = show.description | i18n(locale) %}
 {% set url = show.url %}
 {% set data = show.data %}
-{% set hero = data.hero or null %}
+{% set image = data.hero or null %}
```

Comment:
should we allow for thumbnail here as well?

Reply:
Shows do not have `data.thumbnail` (they do have a `thumbnail`, but it's copied into `data.hero` at the collection level).